src/Uno.UI/Microsoft/UI/Input/InputSystemCursorShapeExtensions.wasm.cs

```diff
@@ -0,0 +1,54 @@
+﻿#if HAS_UNO_WINUI
+
+namespace Microsoft.UI.Input
+{
+	internal static class InputSystemCursorShapeExtensions
+	{
+		internal static string ToCssProtectedCursor(this InputSystemCursorShape inputSystemCursorShape)
```

Comment:
Consider mentioning that a similar method exists in https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/UI/Core/CoreCursorTypeExtensions.wasm.cs

and mention the same in the other file.

Just to ensure that when someone in future edits any of them he knows about the other and make the fix in both.

Reply:
Yeah great idea, thanks @Youssef1313

Reply:
I would even merge them in a single file, probably in the `Uno.UI` namespace (Like `Uno.UI.CursorHelper`).

Reply:
what do you think about this one @Youssef1313 ? Should it be a good idea to have them in a single file?

Reply:
Yeah I think it's a good idea if they can be in a single file.

Reply:
Great thank you both for your feedback, will work on that :) !